src/hooks/sync-sites/use-sync-pull.ts

```diff
@@ -174,28 +169,25 @@ export function useSyncPull( {
 						cancelId: 1,
 					} );
 
-					if ( userChoice === 1 ) {
-						console.log( 'User cancelled pull operation' );
+					const isCanceledByUser = userChoice === 1;
+					if ( isCanceledByUser ) {
```

Comment:
*Wojtek*:
Do we need this variable if it's used only once?

*Vova*:
I just wanted to avoid "magic" number `1`, since at a glance I thought that `1` means kinda `true`, as a confirmation to continue pulling, but actually it's canceling.

So to keep readability, we need either to add comment about `1` or create readable and simple variable. And the good variable name is much better than any comment.

Reply:
What if we renamed `userChoice` to `userCanceled`? Wouldn't it be clear enough without introducing a variable for this simple condition?

Reply:
Oh, my bad, I just realized that I took a look at the code not thoroughly enough, sorry.

We can go with either `userCanceled` or [this way](https://github.com/Automattic/studio/pull/1002/commits/d54e9c4664b2dc4dedd98f6636d2e5c2258ffb7b). I am ok with both options.

Reply:
The way with named constant looks great!